app/packages/core/src/components/Actions/Options.tsx

```diff
@@ -216,19 +216,20 @@ const DynamicGroupsViewMode = ({ modal }: { modal: boolean }) => {
   );
 };
 
-const Lightning = () => {
+const QueryPerformance = () => {
   const [threshold, setThreshold] = useRecoilState(fos.lightningThreshold);
   const config = useRecoilValue(fos.lightningThresholdConfig);
   const reset = useResetRecoilState(fos.lightningThreshold);
   const count = useRecoilValue(fos.datasetSampleCount);
   const theme = useTheme();
+  const enableQpMode = useRecoilValue(fos.enableQueryPerformanceConfig) && useRecoilValue(fos.defaultQueryPerformanceConfig);
```

Comment:
In React, hooks [cannot be run conditionally](https://dev.to/lexlohr/conditional-hooks-1fpj). This should be changed to two separate lines

Reply:
Done ✅